conf/messages.en

```diff
@@ -107,79 +112,186 @@ sectionCompletedState.no = No, I’ll come back to it later
+turnoverIncomeAmount.l4.agent = money owed to your client for work already done
+turnoverIncomeAmount.subHeading.individual = What was your turnover?
+turnoverIncomeAmount.subHeading.agent = What was your client’s turnover?
+turnoverIncomeAmount.checkYourAnswersLabel = TurnoverIncomeAmount
+turnoverIncomeAmount.error.required.individual = Enter your turnover
+turnoverIncomeAmount.error.required.agent = Enter your client’s turnover
+turnoverIncomeAmount.error.nonNumeric.individual = Your turnover must be a number
+turnoverIncomeAmount.error.nonNumeric.agent = Your client’s turnover must be a number
+turnoverIncomeAmount.error.lessThanZero.individual = Your turnover must be more than £0
+turnoverIncomeAmount.error.lessThanZero.agent = Your client’s turnover must be more than £0
+turnoverIncomeAmount.error.overMax.individual = Your turnover must be less than £100,000,000,000
+turnoverIncomeAmount.error.overMax.agent = Your client’s turnover must be less than £100,000,000,000
+turnoverIncomeAmount.change.hidden = Change your turnover income amount
 
-otherIncomeAmount.title = OtherIncomeAmount
-otherIncomeAmount.heading = OtherIncomeAmount
+## Any other income
+anyOtherIncome.title.individual = Is there any other self-employment income you need to tell us about?
+anyOtherIncome.title.agent = Is there any other self-employment income your client needs to tell us about?
+anyOtherIncome.checkYourAnswersLabel = AnyOtherIncome
+anyOtherIncome.error.required.individual = Select yes if there’s any other self-employment income you need to tell us about
+anyOtherIncome.error.required.agent = Select yes if there’s any other self-employment income your client needs to tell us about
+anyOtherIncome.change.hidden = Change any other income
+
+## Other income amount
+otherIncomeAmount.title.individual = How much other self-employment income did you get?
+otherIncomeAmount.title.agent = How much other self-employment income did your client get?
 otherIncomeAmount.checkYourAnswersLabel = OtherIncomeAmount
-otherIncomeAmount.error.nonNumeric = Enter your otherIncomeAmount using numbers
-otherIncomeAmount.error.required = Enter your otherIncomeAmount
-otherIncomeAmount.error.outOfRange = OtherIncomeAmount must be between {0} and {1}
-otherIncomeAmount.change.hidden = OtherIncomeAmount
+otherIncomeAmount.error.required.individual = Enter the amount of other self-employment income you got
+otherIncomeAmount.error.required.agent = Enter the amount of other self-employment income your client got
+otherIncomeAmount.error.nonNumeric.individual = The amount of other self-employment income you got must be a number
+otherIncomeAmount.error.nonNumeric.agent = The amount of other self-employment income your client got must be a number
+otherIncomeAmount.error.lessThanZero.individual = The amount of other self-employment income you got must be more than £0
+otherIncomeAmount.error.lessThanZero.agent = The amount of other self-employment income you got must be more than £0
```

Comment:
Should this message contain `client` instead of `you`?

Reply:
Thank you